### PR TITLE
fixes #3141: SmartPrpxyAuth no longer calls #render_403 method defined in ApplicationController

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -231,6 +231,11 @@ def render_403
     false
   end
 
+  # this is only used in hosts_controller (by SmartProxyAuth module) to render 403's
+  def render_error(msg, status)
```
Comment (skottler) [inline app/controllers/application_controller.rb:235]: The parameters on the method don't get passed along to render_403 so are they needed?

Comment (dmitri-d) [inline app/controllers/application_controller.rb:235]: this is to match render_error method signature defined in Api::BaseController

Comment (skottler) [inline app/controllers/application_controller.rb:235]: Gotcha. :+1:

**Comment by skottler** [comment]
Merged as dd1d641eaa6cc358d80e1372b2ada0382996aadf. Thanks!

**Comment by domcleal** [comment]
No tests?

**Comment by skottler** [comment]
:8ball: 

Would you mind opening another PR with tests @witlessbird? Sorry I didn't enforce that rule @domcleal.

**Pull request closed without merging**